### PR TITLE
WT3 Seawater onshore intake

### PR DESCRIPTION
**Pull request by adam-a-a**
## Summary/Motivation:
SW onshore intake for Santa Barbara case study.

## Changes proposed in this PR:
- add unit and tests
- modified yaml

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by luohezhiming — DISMISSED** [review]

**Review by bknueven — DISMISSED** [review]

**Review by bknueven — DISMISSED** [review]

**Review by bknueven — DISMISSED** [review]

File: watertap/unit_models/zero_order/sw_onshore_intake_zo.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+This module contains a zero-order representation of seawater onshore intake unit.
+operation.
```
Comment (luohezhiming) [inline watertap/unit_models/zero_order/sw_onshore_intake_zo.py:15]: Probably delete the period after "seawater onshore intake unit" in the docstring.

**Review by luohezhiming — CHANGES_REQUESTED** [review]

File: watertap/data/techno_economic/sw_onshore_intake.yaml [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
-    units: dimensionless
-    reference: Intake and Pump Costs from Voutchkov (see capital cost curves in VAR).
-      Energy consumption value based on 100-ft head pump and is consistent with reported
-      Carlsbad data.
-  default_removal_frac_mass_solute:
-    value: 0
+default:  # needed some parameters as placeholder, so including cost params here
+  cap_basis_par:
+    value: 0.000215
+    units: M$       # TODO: unsure of the units
```
Comment (bknueven) [inline watertap/data/techno_economic/sw_onshore_intake.yaml]:
```suggestion
    units: M$
```

File: watertap/unit_models/zero_order/sw_onshore_intake_zo.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+This module contains a zero-order representation of seawater onshore intake unit.
```
Comment (bknueven) [inline watertap/unit_models/zero_order/sw_onshore_intake_zo.py]:
```suggestion
This module contains a zero-order representation of seawater onshore intake unit
```

**Review by kurbansitterley — APPROVED** [review]
LGTM

**Review by luohezhiming — APPROVED** [review]

**Pull request merged**